samples/uhttp_sample/uhttp_sample.c

```diff
@@ -64,19 +65,42 @@ static void on_closed_callback(void* callback_ctx)
     printf("Connection closed callback\r\n");
 }
 
-static void test_http_get(void)
+static HTTP_CLIENT_HANDLE create_uhttp_client_handle(HTTP_SAMPLE_INFO* sample_info, const char* host_name, int port_num)
 {
     SOCKETIO_CONFIG config;
+    TLSIO_CONFIG tls_io_config;
+    const void* xio_param;
+    const IO_INTERFACE_DESCRIPTION* interface_desc;
+    if (port_num == HTTPS_PORT_NUM)
+    {
+        tls_io_config.hostname = host_name;
+        tls_io_config.port = port_num;
+        tls_io_config.underlying_io_interface = NULL;
+        tls_io_config.underlying_io_parameters = NULL;
+        xio_param = &tls_io_config;
+        // Get the TLS definition
+        interface_desc = platform_get_default_tlsio();
+    }
+    else
```

Comment:
Might help to add a comment here that the port_num == HTTP_PORT_NUM (because those are the only two options in this sample)